public/src/components/server_config/settings.vue

```diff
@@ -57,8 +57,8 @@ export default {
                 {value: "FreeForAll", label: "Mode FreeForAll"},
                 {value: "GT3", label: "Mode GT3"},
                 {value: "GT4", label: "Mode GT4"},
-                {value: "Cup", label: "Mode CUP"},
-                {value: "ST", label: "Mode ST"}
+                {value: "GTC", label: "Mode GTC"},
+
```

Comment:
I am not the best in JS... but as this is the last element in the array, the line can end up with `,` ???

Reply:
Safe in actual objects (like this), not safe in JSON.